enterprise_subsidy/apps/api/v1/urls.py

```diff
@@ -1,4 +1,30 @@
-""" API v1 URLs. """
+"""
+URL definitions for enterprise-subsidy API version 1.
+
+Listing, reading, and testing subsidies::
+
+  GET /api/v1/subsidies/?enterprise_customer_uuid={uuid}&subsidy_type={"learner_credit","subscription"}
```

Comment:
do we want to make `enterprise_customer_uuid` a path param rather than a query param? seems more restful to me, ie `/api/v1/subsidies/<enterprise_customer_uuid>/<subsidy_uuid>?subsidy_type=foo`